rflx/pyrflx/typevalue.py

```diff
@@ -169,11 +182,13 @@ class EnumValue(ScalarValue):
     def __init__(self, vtype: Enumeration) -> None:
         super().__init__(vtype)
 
-    def assign(self, value: str, check: bool = True) -> None:
+    def assign(self, value: str, length: int = 0, check: bool = True) -> None:
+        assert isinstance(self._type, Enumeration)
```

Comment:
`assert isinstance(self._type, Enumeration)` should not be needed here.

Reply:
Fixed by 9f39fdc